Make.inc

```diff
@@ -239,6 +239,8 @@ BUILD_LLVM_CLANG := 0
 BUILD_LLDB := 0
 USE_POLLY := 0
 USE_POLLY_OPENMP := 0
+USE_POLLY_ACC := 0
+CUDALIB_INCLUDE_DIR := /usr/local/cuda/include
```

Comment:
If CUDA supports Windows, this path will be incorrect. Is there a way to detect this rather than having to hard-code it?